### ISSUE TITLE
JOSS

### ISSUE THREAD

**Issue by arosen93** [chemics/chemics#6]
Hey! I'm submitting this fake issue per our prior conversation. If you submit this to JOSS one day, feel free to ping me (or suggest me as a reviewer directly). Would be happy to review this.

Nice work on this!

Cheers,
Andrew

**Comment by wigging** [comment]
How does JOSS handle new releases, do I need to submit to JOSS for each new version of the Chemics package?

**Comment by arosen93** [comment]
No, generally new releases are not submitted to JOSS. The short manuscript that goes alongside the code should describe the overall structure of the package and its purpose, which generally doesn't drastically change between versions. There's a link to an archived version of the package (just in case you, for instance, decide to delete everything on GitHub), but the manuscript links to the current version of the package and is not a problem.
